packages/components/src/tripTypeButton/__tests__/TripTypeButton.test.js

```diff
@@ -16,7 +16,7 @@ const { getByType } = render(
 
 it('should execute onPress method', () => {
   fireEvent(getByType(TouchableWithoutFeedback), 'press');
-  expect(onPress).toHaveBeenCalled();
+  expect(onPress).toHaveBeenCalledWith(undefined);
```

Comment:
*Just for my information*
Why do you change `toHaveBeenCalled()` into `toHaveBeenCalledWith(undefined)`?

Reply:
because we have warnings from eslint -> `jest/prefer-called-with`

Reply:
thanks